packages/zowe-explorer/CHANGELOG.md

```diff
@@ -23,6 +24,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Fixed issue where Zowe Explorer would present the "No configs detected" notification when initialized in a workspace without a Zowe team configuration. [#3280](https://github.com/zowe/zowe-explorer-vscode/issues/3280)
 - Reduced the number of MVS API calls performed by `vscode.workspace.fs.readFile` when fetching the contents of a data set entry. [#3278](https://github.com/zowe/zowe-explorer-vscode/issues/3278)
 - Fixed an issue to review inconsistent capitalization across translation strings. [#2935](https://github.com/zowe/zowe-explorer-vscode/issues/2935)
+- Updated dependecies for technical currency. [#3786](https://github.com/zowe/zowe-explorer-vscode/issues/3786)
```

Comment:
```suggestion
- Updated dependencies for technical currency. [#3786](https://github.com/zowe/zowe-explorer-vscode/issues/3786)
```

Reply:
Resolved in 172cfe573ba7cff7df22d3b3c4cdf4e9fc773e81